MainModule/Server/Commands/Fun.lua

```diff
@@ -925,7 +925,7 @@ return function(Vargs, env)
 							v.Character.Pants:Destroy()
 						end
 						for _, m in v.Character:GetChildren() do
-							if m:IsA("BasePart") and prt.Name ~= "HumanoidRootPart" then
+							if m:IsA("BasePart") and m.Name ~= "HumanoidRootPart" then
```

Comment:
![image](https://github.com/Epix-Incorporated/Adonis/assets/90981615/9d17c987-e1d3-4981-9302-7057d47dc2cf)

Reply:
I left my pc and when I got back I got jumpscared by this lol